.github/workflows/publishes.yml

```diff
@@ -0,0 +1,57 @@
+name: "Publishes"
+
+on:
+  push:
+    branches: [ snapshot, 1.*.*, 2.*.* ]
+    #tags: [ "*" ]
+
+jobs:
+  publish:
+    name: "Publish"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        type: [ "OSSRH", "image" ]
+    steps:
+      # step 1
+      - name: "Checkout"
+        uses: actions/checkout@v2.4.0
+
+      # step 2
+      - name: "Setup Java JDK"
+        uses: actions/setup-java@v2.5.0
+        with:
+          distribution: 'zulu'
+          java-version: 8
+          server-id: ossrh
+          server-username: OSSRH_USERNAME
```

Comment:
why need `server-username`?

Reply:
This `server-username` will set to the `settings.xml` for `maven-deploy-plugin`